Week2/homework/js-exercises/3-recipe-card.js

```diff
@@ -0,0 +1,14 @@
+﻿'use strict';
+
+var recipe = {
+    'title': 'Mole',
+    'servings': 2,
+    'ingredients': ['cumin', 'cinnamon', 'cocoa']
+};
+
+console.log(recipe.title);
+console.log('Servings: ' + recipe.servings);
+console.log('Ingredients:');
+for (var i = 0; i < recipe.ingredients.length; i++) {
+    console.log(recipe.ingredients[i]);
+}
```

Comment:
I would suggest also looking into `Object.keys` method in JavaScript

Reply:
Also, `let` instead of `var` please

Reply:
thank you so much Yash-kapila for the feedback an I will change the var to let.